core_impl/src/main/java/io/opencensus/stats/MutableViewData.java

```diff
@@ -127,6 +130,7 @@ void record(StatsContextImpl tags, long value) {
    * Convert this {@link MutableViewData} to {@link ViewData}.
    */
   ViewData toViewData(Clock clock) {
+    removeExpiredValues(clock.now());
```

Comment:
We should also remove expired values when we add new values, to limit the queue size in the case where getView is never called.